website/content/docs/backends-overview.md

```diff
@@ -18,6 +18,7 @@ Individual backends should provide their own configuration documentation, but th
 | `site_domain`   | `location.hostname` (or `cms.netlify.com` when on `localhost`) | Sets the `site_id` query param sent to the API endpoint. Non-Netlify auth setups will often need to set this for local development to work properly. |
 | `base_url`      | `https://api.netlify.com` (GitHub, Bitbucket) or `https://gitlab.com` (GitLab)                                     | OAuth client hostname (just the base domain, no path). **Required** when using an external OAuth server or self-hosted GitLab.                               |
 | `auth_endpoint` | `auth` (GitHub, Bitbucket) or `oauth/authorize` (GitLab)                  | Path to append to `base_url` for authentication requests. Optional.                                                                                  |
+| `include_cookies_in_requests` | `false` | Whether or not to include cookies in XHR requests to the git gateway. You should only change this if you are self-hosting the git gateway and need cookies. |
```

Comment:
How about renaming the option to `include_credentials_in_api_requests ` to correlate with the `fetch` API?
Also, I would phrase it a bit differently:

> Git Gateway only. Sets the [credentials mode](https://fetch.spec.whatwg.org/#concept-request-credentials-mode) for API requests to `include` which results in including cookies in those requests. Useful when self hosting Git Gateway with a custom authentication method.